packages/sdk/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.2.7
```

Comment:
Please double check with release manager. The version is 0.3.0.

Reply:
Corrected, thanks